package.json

```diff
@@ -279,5 +283,11 @@
     "blocking": false,
     "logInfo": false,
     "logError": true
+  },
+  "lint-staged": {
+    "*.{js,ts,tsx}": [
+      "eslint --cache --fix",
```

Comment:
I'm not a use fan of linting in a commit-hook. It can be frustrating for developers when there are issues getting the tooling set up, or WIP that does not yet lint. Can we put this in a CI check instead?

Reply:
yeah for sure! i'll leave it out of the the pre-commit config

Reply:
just to clarify, are you just referring to the `eslint` step here or `lint-staged` as a whole?